openedx/core/djangoapps/waffle_utils/testutils.py

```diff
@@ -2,66 +2,18 @@
 Test utilities for waffle utilities.
 """
 
-
-from waffle.testutils import override_flag
+# Import from edx-toggles to preserve import paths
+# TODO: Deprecate and remove
```

Comment:
Interesting that you did this here, but not the non-test functionality.  What is the plan for knowing when these are no longer used?  Don't we want to remove these fully in this PR to ensure all tests pass, or are you making multiple passes?

Reply:
> Interesting that you did this here, but not the non-test functionality.

What functionality are you referring to? The changes in this PR affect only test-related features.

>  What is the plan for knowing when these are no longer used? Don't we want to remove these fully in this PR to ensure all tests pass, or are you making multiple passes?

I think you're right: I initially wanted to remove these imports but later forgot about them. I'll remove them now.

Reply:
Removing edx-platform's `WaffleFlag`, `WaffleSwitch`, etc., removes more than test functionality.

I agree that removing test only functionality (like this) without first deprecating should be fine.

Reply:
> Removing edx-platform's WaffleFlag, WaffleSwitch, etc., removes more than test functionality.

I don't think it does -- not in this PR, because the classes are still available as imports from `edx_toggles.toggles` in waffe_utils.__init__ : https://github.com/edx/edx-platform/pull/25417/files#diff-fb4dfb16a208685aa89aef3ea799df781f5794043a782e4873872e3bdeb4b3f8R9

Reply:
I had missed the import.  Thanks.  We just need to add the wrapper to set the custom attribute.  Thanks!